### DISCUSSION TITLE
怎么根据App BundleId来launchApp

### DISCUSSION THREAD

**Discussion by magicdoraemon** [liyu158163/new_cartool#3] [Q&A] [answered]
怎么根据App BundleId来launchApp

**Comment by fisher158163** [comment, marked as answer]
// MARK: 根据App BundleId来launchApp
    class func launchApp(BundleId: String) {
        NSWorkspace.shared.launchApplication(withBundleIdentifier: BundleId, options: .withoutActivation, additionalEventParamDescriptor: nil, launchIdentifier: nil)
    }
